### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by candancelik**

### CODE REVIEW

File: dde-store_tr.ts [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+        </message>
+        <message>
+            <location filename="../pages/categorypage.cpp" line="19"/>
+            <source>Ratings</source>
+            <translation>Derecelendirmeye göre</translation>
+        </message>
+        <message>
+            <location filename="../pages/categorypage.cpp" line="39"/>
+            <source>No results for %1</source>
+            <translation>%1 için sonuç bulunamadı/translation>
```
Comment (ghost) [inline dde-store_tr.ts]: Missing the < in the closing </translation> tag.

File: README.tr.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<h3 align="center">DTK ile oluşturulmuş DDE için bir uygulama mağazası</h3>
+
+
+## Bu nedir?
+ 
+Bu uygulama mağazası, Deepin Masaüstü Ortamı'nın (Deepin Desktop Environment, DDE) diğer dağıtımlar üzerinde kullanılması amacıyla tasarlanmıştır. Resmi Deepin mağazası Debian ve üzerinde Deepin depolarının bulunduğu bir web görünümü kapsamında inşa edilmiştir, bu da Deepin mağazasını diğer dağıtımlarla uyumsuz hale getirir. Bu mağaza, PackageKit ve AppStream ile oluşturulmuştur, bu nedenle herhangi bir dağıtım üzerinde [almost](#deepin-distro) mükemmel çalışmalıdır!
+
+## Deepin Dağıtımı
+
+Bu sadece resmi Deepin dağıtımı için geçerlidir, genel olarak DDE için geçerli değildir. Deepin, bu uygulamaya gerekli tüm verileri sağlayan AppStream'i desteklemez. **Bu mağaza, Deepin dağıtımında desteklenmiyor.** Bu, yine de DDE'yi çalıştıran diğer dağıtımlar için tasarlandı, bu yüzden onu desteklemeye dair hiçbir planım yok. Deepin AppStream'i kullanmaya başlarsa, düşünebilirim. Deepin kullanıyor ve alternatif bir uygulama mağazası arıyorsanız, [Spark Store] (https://www.spark-app.store) veya [Deepines Store] (https: // deepines. com) gibi diğer harika projelere göz atın!
```
Comment (ghost) [inline README.tr.md]: Spark Store and Deepines Store links are broken. Links need to be formatted like this:
```markdown
[Deepines Store](https://deepines.com)
```
Make sure not to add any extra spaces between the brackets or in the link.

File: dde-store_tr.ts (hunk 1 above)
Comment (candancelik) [reply dde-store_tr.ts]: Corrected. Thank you.

File: README.tr.md (hunk 2 above)
Comment (candancelik) [reply README.tr.md]: Corrected with thanks.

**Review by ghost — APPROVED** [review]
Great, thank you for fixing those!

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Merging, thank you so much for contributing!

**Pull request merged**